res/css/structures/auth/_CompleteSecurity.scss

```diff
@@ -70,6 +70,10 @@ limitations under the License.
     }
 }
 
+.mx_CompleteSecurity_recoveryKeyInput {
+    width: 368px;
```

Comment:
this feels precise enough that it needs a comment to explain why 368 was chosen and not 370. Something like `// dialog width minus 20px padding on either side` (as I presume this is essentially what it is?)